README.md

```diff
@@ -12,7 +11,7 @@ The Hyperledger Fabric CLI is a tool used to interact with [Fabric networks](htt
 ## Installation
 
 1. Clone this repo
-2. Install `gobin` using `GO111MODULE=off go get -u github.com/myitcv/gobin`
+2. Install `gobin` using `GO111MODULE=off go Update README with CI badge and remove gerritget -u github.com/myitcv/gobin`
```

Comment:
It looks like some extra text got copied in?

Reply:
fixed